tests/avalon_test_framework/tests/work_order_tests/test_submit.py

```diff
@@ -616,13 +575,12 @@ def test_workordersubmit_indata_index2randomstr(self):
             env['work_order_submit_input_file'])
 
         assert (
-            check_negative_test_responses(
-                result_response,
-                "Invalid Request")
+            verify_test(
```

Comment:
The test is actually a positive test, refer the bug: https://jira.devtools.intel.com/browse/TCFH-815